src/Raven.Studio/typescript/widgets/virtualGrid/columns/iconsPlusTextColumn.ts

```diff
@@ -0,0 +1,58 @@
+﻿/// <reference path="../../../../typings/tsd.d.ts"/>
+
+import virtualColumn = require("widgets/virtualGrid/columns/virtualColumn");
+import virtualGridController = require("widgets/virtualGrid/virtualGridController");
+
+class iconsPlusTextColumn<T> implements virtualColumn {
+    width: string;
+    header: string;
+
+    private readonly dataForHtml: (obj: T) => iconPlusText[] | string ;
```

Comment:
remove space before ;
btw. do we need `| string` here? Isn't that escape hatch?

Reply:
this `dataForHtml` is can be either:
1. array with icons & text info
2. string with just text for the 'common item' row (i.e. **Indexes** column)

Reply:
ok so let's remove space before `;` and leave this as it is.